sphinx/ext/autodoc/directive.py

```diff
@@ -84,7 +84,12 @@ def process_documenter_options(documenter: "Type[Documenter]", config: Config, o
         else:
             negated = options.pop('no-' + name, True) is None
             if name in config.autodoc_default_options and not negated:
-                options[name] = config.autodoc_default_options[name]
+                if name == "exclude-members":
+                    if config.autodoc_default_options[name]:
+                        options[name] = config.autodoc_default_options[name] \
+                            + options.get(name, '')
```

Comment:
It seems this simply concatenate to the `autodoc_default_options`. Is it working fine? I guess you need to use `','.join(...)` instead. I think adding a testcase will help this.